src/lib/wallet/GoodWallet.js

```diff
@@ -398,7 +399,14 @@ export class GoodWallet {
 
     log.info({ amount })
 
-    const sendLink = `${Config.publicUrl}/AppNavigation/Dashboard/Home?receiveLink=${generatedString}&reason=${reason}`
+    const sendLink = generateShareLink('send', {
+      receiveLink: generatedString,
+      reason
+    })
+
+    if (!sendLink) {
```

Comment:
why don't throw inside generateShareLinnk?

Reply:
Done this change. Updated `Receive` and `ReceiveAmount` also to handle the error.